### ISSUE TITLE
RxD and TxD interchanged

### ISSUE THREAD

**Issue by weigu1** [MHEtLive/ESP32-MINI-KIT#7]

**Comment by weigu1** [comment]
Hi,

The MHEtLive ESP32MiniKit is not compatible with the LOLIN/WEMOS D1 mini pro. RxD and TxD are not on the same pins but interchanged. You can see this in the circuit diagram. The pinout picture I found in the net is wrong.
TxD is near GPIO22 and RxD is the outer pin!

**Comment by petus** [comment]
Yes, that's true. I just develop my own design based on MiniKit.
The schematic is ok.
